Dockerfile

```diff
@@ -1,15 +1,13 @@
-# docker-logentries
-#
-# VERSION 0.2.0
+# r7insight_docker
 
 FROM node:0.12-onbuild
-MAINTAINER Rapid 7 - Logentries <support@logentries.com>
+LABEL maintainer="Rapid 7 - Platform Support <platformsupport@rapid7.com>"
```

Comment:
Is `platformsupport` the best address to use?

Reply:
Asked @cheffernan-r7 and he said use this one